### PR TITLE
[EasyDoctrine] Add IntegerNumberType

### PR DESCRIPTION
**Pull request by alexndlm**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

**Review by roman-eonx — CHANGES_REQUESTED** [review]

**Review by roman-eonx — CHANGES_REQUESTED** [review]

File: packages/EasyUtils/src/Validator/Constraints/AbstractNumberComparisonValidator.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyUtils\Validator\Constraints;
```
[the remark this answers is not in the document]
Comment (roman-eonx) [reply packages/EasyUtils/src/Validator/Constraints/AbstractNumberComparisonValidator.php:5]: OK, let's follow the Symfony way here. And please ignore `\EonX\EasyCore\Bridge\Symfony\Validator\Constraints\IntegerValidator`, our intention is to get rid of EasyCore soon.

File: packages/EasyUtils/src/Serializer/IntegerNumberNormalizer.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyUtils\Serializer;
```
[the remark this answers is not in the document]
Comment (alexndlm) [reply packages/EasyUtils/src/Serializer/IntegerNumberNormalizer.php]: Done

File: packages/EasyUtils/src/ValueObject/Number.php [hunk 3]
```diff
@@ -0,0 +1,206 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyUtils\ValueObject;
```
[the remark this answers is not in the document]
Comment (alexndlm) [reply packages/EasyUtils/src/ValueObject/Number.php]: Done

File: packages/EasyUtils/src/Validator/Constraints/AbstractNumberComparisonValidator.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexndlm) [reply packages/EasyUtils/src/Validator/Constraints/AbstractNumberComparisonValidator.php:5]: Done

File: packages/EasyUtils/src/Types/IntegerNumberType.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+declare(strict_types=1);
+
+namespace EonX\EasyUtils\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\BigIntType as DoctrineBigIntType;
+use Doctrine\DBAL\Types\ConversionException;
+use EonX\EasyUtils\ValueObject\Number;
+
+final class IntegerNumberType extends DoctrineBigIntType
```
[the remark this answers is not in the document]
Comment (alexndlm) [reply packages/EasyUtils/src/Types/IntegerNumberType.php]: Done

**Review by roman-eonx — APPROVED** [review]

**Review by roman-eonx — APPROVED** [review]

**Review by akl-95 — APPROVED** [review]

**Review by natepage — APPROVED** [review]

**Pull request merged**